### PR TITLE
Remove the three wakeup-related settings

### LINKED ISSUE #684: Kafka 2 polling: Remove unused settings
The wakeup handling goes away with the new `poll(Duration)` method, thus
* wakeup-timeout
* max-wakeups
* wakeup-debug
have no effect anymore and should be removed.

### PR DESCRIPTION
**Pull request by ennru**
Fixes #684

### CODE REVIEW

**Comment by ennru** [comment]
Failure is #683

File: core/src/main/resources/reference.conf [hunk 1]
```diff
@@ -54,29 +54,19 @@ akka.kafka.consumer {
...
-  # DEPRECATED: No op as waking up consumer is not used anymore
-  wakeup-timeout = 3s
-
-  # After exceeding maximum wakeups the consumer will stop and the stage will fail.
-  # Setting it to 0 will let it ignore the wakeups and try to get the polling done forever.
-  # DEPRECATED: No op as waking up consumer is not used anymore
-  max-wakeups = 10
+
+  # Not used anymore (since 1.0-RC1)
+  # wakeup-timeout = 3s
```
Comment (2m) [inline core/src/main/resources/reference.conf]: Should we just remove these altogether?

Comment (ennru) [reply core/src/main/resources/reference.conf]: My thinking was to leave them (during 1.0.x) so people looking at their existing configs understand that they may remove them.

Comment (2m) [reply core/src/main/resources/reference.conf]: Alright.

**Pull request merged**